assets/css/custom.css

```diff
@@ -1015,3 +1015,13 @@ h2.gray-underline:after {
 .client-links a {
     padding: 20px;
 }
+
+/*Post with text side by side images*/
+.image-side-by-side-text {
+	float:left;
+	margin: 0px 50px 15px 20px;
+}
+
+.text-side-by-side-image {
+	margin: 0px 50px 0px 0px;
```

Comment:
Can we add a min-width here? Otherwise the text can end up being in a very small 10px columns on mobile and it gets unreadable (had that on a simulation of Nexus 5X in CHrome)